app/mailers/reminder_mailer.rb

```diff
@@ -1,36 +1,45 @@
 # frozen_string_literal: true
 
 class ReminderMailer < ApplicationMailer
```

Comment:
For each of the mailer methods we are sending in objects (user, carrier, etc.) and then pulling out one or two bits of information from them based on knowledge of their structure. We might consider just sending in the information needed rather than objects.